src/kava.ts

```diff
@@ -875,16 +875,20 @@ class Kava extends BasePlugin {
   }
 
   private _getPosition(): number {
+    let currentTime = this.player.currentTime;
```

Comment:
what does getposition do ? is it possible to use normalizedCurrentTime for live ? because i don't know how common it is for live to have duration

Reply:
getPosition calculate the time in the video when the analytics request is sent and send this position in the request. i can keep the regular currentTime in live and not add the seekTo value in live